### ISSUE TITLE
Does it work?

### ISSUE THREAD

**Issue by Ytoblo** [Mark-225/DynmapSpoofer#1]
an interesting idea, I would not want her to die

**Comment by Mark-225** [comment]
So far it doesn't work. I temporarily "abandoned" this plugin a few moths ago because while they seem similar on the surface, bluemap and dynmap have some deeply rooted structural differences in their marker implementations that I can't just write a simple translator for. I would need a lot of config options to decide what to do in situations where it isn't really obvious how to translate dynmap API calls to bluemap and back then I just didn't have the time for that.
But thanks for reminding me about this project as I have a lot more time on my hands for the next month or so and might continue working on this.
In case I don't find the time or give up again, feel free to just steal my idea :D

**Comment by Ytoblo** [comment]
...

**Comment by Mark-225** [comment]
Yeah, I declare this project dead for now :D  I started it when there was basically no 3rd party support for Bluemap and also more as a proof of concept rather than anything production ready. There is now a lot of support for many different region and marker plugins and it kinda isn't worth all the effort anymore...
Maybe I'll finish it at some point for fun, but for now I'd recommend checking the 3rd party support list on BlueMap's Discord or suggesting BlueMap support to the developers of whatever plugins you want to use.
